src/error.rs

```diff
@@ -32,6 +32,8 @@ pub enum ResponseError {
     /// Abort is for user to indicate that the state can be dropped;
     /// if received by routing, it will drop the state.
     Abort,
+    /// On low balance or no account registered
+    LowBalance(Data, u32),
     /// invalid request
```

Comment:
I though NoAccount was agreed to be put in , though it won't be hit during the test?